python/pyarrow/device.pxi

```diff
@@ -137,9 +137,7 @@ cdef class MemoryManager(_Weakrefable):
         return self.memory_manager
 
     def __repr__(self):
-        return "<pyarrow.MemoryManager device: {}>".format(
-            frombytes(self.memory_manager.get().device().get().ToString())
-        )
+        return f"<pyarrow.MemoryManager device: {frombytes(self.memory_manager.get().device().get().ToString())}>"
```

Comment:
This line is a bit too long, perhaps use a temporary variable?

Reply:
Resolved in [558be8d](https://github.com/apache/arrow/pull/45629/commits/558be8d3559a1ed3441cc57cec343d6889fff5f0).